magento/magento2
# Issue #19417 Fixed

Products added to a category are not shown at the frontend

### Description (*)

Assign a product to category, and schedule cron jobs, Product was not listing in category after running indexer via cron. Fixed now in this PR.
### Fixed Issues (if relevant)

1. #19417 
2. ...

### Manual testing scenarios (*)

1. Run magento indexer:reindex to make all indexes up to date.
2. Switch all indexes to the "Update Scheduled" mode.
3. Set up Cron.
4. Create a simple product (visible at the frontend, qty > 0)
5. Create a category.
6. Go to the frontend and make sure that the category is empty.
7. Open the created category (step 4) and assign the created product to it.
8. Save category changes.

Wait until Cron reindexes category changes or run magento indexer:reindex catalog_category_product.
Open the category at the frontend. Now products are listing.

### Contribution checklist (*)
